### PR TITLE
Fix linter errors

### PR DESCRIPTION
**Pull request by ColdForeign**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-# Don't check these into the repo as LF to work around TeamCity bug
-*.xml     -text
+* text=auto eol=crlf
+
+# Declare files that will always have LF line endings on checkout.
+*.sh text eol=lf
+
+# Don't check these into the repo as LF to work around TeamCity bug
+*.xml     -text
 *.targets -text
```
Comment (MaxymGorn) [inline .gitattributes:8]: add end line

File: src/Cropper.Blazor/Client/Pages/Crop/CropPage.razor [hunk 2]
```diff
@@ -25,6 +25,10 @@
                 "polygon image"
             })" />
 
+
+
+
```
Comment (MaxymGorn) [inline src/Cropper.Blazor/Client/Pages/Crop/CropPage.razor:30]: remove lines plz

File: src/Cropper.Blazor/Client/Components/Docs/DocsApi.razor.cs [hunk 3]
```diff
@@ -200,10 +200,10 @@ private bool CheckIsTwoWayProperty(PropertyInfo propertyInfo)
         {
             PropertyInfo? eventCallbackInfo = Type.GetProperty(propertyInfo.Name + "Changed");
 
-            return eventCallbackInfo != null &&
-                   eventCallbackInfo.PropertyType.Name.Contains("EventCallback") &&
-                   eventCallbackInfo.GetCustomAttribute<ParameterAttribute>() != null &&
-                   eventCallbackInfo.GetCustomAttribute<ObsoleteAttribute>() == null;
+            return eventCallbackInfo != null &&
```
Comment (MaxymGorn) [inline src/Cropper.Blazor/Client/Components/Docs/DocsApi.razor.cs:203]:
```suggestion
            return eventCallbackInfo != null &&
```

File: src/Cropper.Blazor/Client/Pages/Crop/CropPage.razor (hunk 2 above)
Comment (ColdForeign) [reply src/Cropper.Blazor/Client/Pages/Crop/CropPage.razor]: removed

File: src/Cropper.Blazor/Client/Components/Docs/DocsApi.razor.cs (hunk 3 above)
Comment (ColdForeign) [reply src/Cropper.Blazor/Client/Components/Docs/DocsApi.razor.cs]: fixed

**Review by MaxymGorn — APPROVED** [review]

**Review by MaxymGorn — APPROVED** [review]

**Review by ColdForeign — COMMENTED** [review]
🎖

**Review by MaxymGorn — APPROVED** [review]

**Pull request merged**